### ISSUE TITLE
[Enchantment] Server management without restart

### ISSUE THREAD

**Issue by Spontini** [PaperMC/Waterfall#474]
Hello, i don't know why this feature is not exist in the Bungeecord/ Waterfall yet but i think we should've commands to do these things;

- Create/ Remove Servers, ForcedHosts, Change MOTD
- Create/ Remove Groups, Permissions and change group of the user.

**Comment by LB45440078L** [comment]
Yes it would be nice to see this in-game , the only way to do this currently is using external plugins.

**Comment by Spontini** [comment]
> Yes it would be nice to see this in-game , the only way to do this currently is using external plugins.

Yes and sadly most of the plugins are not that optimized, i believe the real creators can do a lot better job.

**Comment by electronicboy** [comment]
> Create/ Remove Servers, ForcedHosts, Change MOTD

Waterfall exposes API for the former, the others can already be done by a plugin;
Not really sure that there is much interest in designing commands for this, etc

> Create/ Remove Groups, Permissions and change group of the user

The mechanism built into bungeecord is trivial and flawed, I do not consider the built in mechanism as being suitable for production servers as-is, use a permissions plugin which is actually designed to be robust

**Comment by Spontini** [comment]
> > Create/ Remove Servers, ForcedHosts, Change MOTD
> 
> Waterfall exposes API for the former, the others can already be done by a plugin;
> Not really sure that there is much interest in designing commands for this, etc
> 
> > Create/ Remove Groups, Permissions and change group of the user
> 
> The mechanism built into bungeecord is trivial and flawed, I do not consider the built in mechanism as being suitable for production servers as-is, use a permissions plugin which is actually designed to be robust

Plugins are exist yes but most of them are outdated or has bunch of bugs, the permission side is alright i mean probably everyone is using such plugins like LuckPerms but Creating/ Removing servers with ForcedHosts etc always been a issue without a restart. Especially i couldn't find any single plugin that can create ForcedHost
